### DISCUSSION TITLE
Where can we start to build Microsoft Teams apps?

### DISCUSSION THREAD

**Discussion by zawhtut** [microsoft/hack-together-teams#27] [Ideas]
Just in case we are not sure about where to start the MS Teams app building, we can start by following this tutorial...

https://learn.microsoft.com/en-us/microsoftteams/platform/sbs-botbuilder-conversation-ai

~ Cheers

**Comment by waldekmastykarz** [comment]
Thanks for sharing @zawhtut. We've also put together some more resources that show other types of apps for Microsoft Teams on the readme @ https://github.com/microsoft/hack-together-teams#recommended-learning-materials-.
